src/main/java/pl/marekjava/App.java

```diff
@@ -61,7 +61,7 @@ private static Integer getNumberOfPlayers() {
 
         if (numberOfPlayers < 2 || numberOfPlayers > 4) {
             System.out.println("Ilość graczy powinna zmieścić się pomiędzy 2 a 4.");
```

Comment:
Tutaj mimo wszystko zamiast zwracać -1 proponuję rzucić wyjątek - IllegalStateException albo IllegalArgumentException, bo to książkowy przykład na zastosowanie wyjątków.
```
if (numberOfPlayers < 2 || numberOfPlayers > 4) {
            throw new IllegalArgumentException("Ilość graczy powinna zmieścić się pomiędzy 2 a 4.");
```

I potem w main zamaist tego if'a dać

```
try {
 Integer numberOfPlayers = getNumberOfPlayers();
} catch (IllegalArgumentException ex) 
  System.out.println(ex.getMessage());
 return;
}
```

Bo zwracanie -1 to trochę niższy poziom, gdy operujemy blisko systemu operacyjnego. W Javie zdecydowanie lepiej używać wyjątków